include/dem/particle_wall_contact_force.h

```diff
@@ -496,30 +496,28 @@ class ParticleWallContactForce
   {
     using namespace Parameters::Lagrangian;
 
-    if constexpr (rolling_friction_model ==
-                  RollingResistanceMethod::no_resistance)
+    if constexpr (rolling_friction_model == RollingResistanceMethod::none)
       {
         return no_rolling_torque();
       }
 
-    if constexpr (rolling_friction_model ==
-                  RollingResistanceMethod::constant_resistance)
+    if constexpr (rolling_friction_model == RollingResistanceMethod::constant)
       {
         return constant_rolling_torque<PropertiesIndex>(particle_radius,
                                                         particle_properties,
                                                         rolling_friction_coeff,
                                                         normal_force_norm);
       }
 
-    if constexpr (rolling_friction_model == viscous_resistance)
+    if constexpr (rolling_friction_model == viscous)
       {
         return viscous_rolling_torque<PropertiesIndex>(particle_radius,
                                                        particle_properties,
                                                        rolling_friction_coeff,
                                                        normal_force_norm,
                                                        normal_unit_vector);
       }
-    if constexpr (rolling_friction_model == epsd_resistance)
+    if constexpr (rolling_friction_model == epsd)
```

Comment:
These ones why don't they have the RollingResistanceMethod:: 
prefix?

Let's try to make it uniform (I know it was already like this before)